pytripgui/controller/tree_menu_cont.py

```diff
@@ -157,8 +157,8 @@ def add_field(self):
         logger.debug("add_field_new() {}".format(None))
 
         from pytrip.tripexecuter import Field
-        from pytripgui.field_editor import FieldQtView
-        from pytripgui.field_editor import FieldController
+        from pytripgui.field_vc import FieldQtView
+        from pytripgui.field_vc import FieldController
```

Comment:
Just a question, what does "_vc" stand for?

Reply:
view controller i assume. I am still sleeping.

Reply:
yes, it is "view controller"